h3i/src/frame.rs

```diff
@@ -427,26 +431,3 @@ impl<'a> Serialize for SerializableQFrame<'a> {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
-    use super::*;
-
-    #[test]
-    fn test_status_code() {
```

Comment:
should we update the test case instead of removing it?

Reply:
There's not much remaining to test, but moar tests is better than less tests I guess :shrug:

Reply:
Just noticed that doc test is executed in quiche repo. I'm ok with it.
https://github.com/cloudflare/quiche/blob/master/.github/workflows/nightly.yml#L36